src/main/java/org/apache/hadoop/hive/serde2/avro/AvroSerdeUtils.java

```diff
@@ -276,28 +276,30 @@ public static HiveDecimal getHiveDecimalFromByteBuffer(ByteBuffer byteBuffer, in
     return dec;
   }
 
+  private static Schema.Parser getSchemaParser() {
```

Comment:
> Patch adapted slightly from 4dd2072e8ccd79a6dbe1cadde043a22d9d4efbea.

this doens't resolve. replace with a link: https://github.com/apache/hive/commit/4dd2072e8ccd79a6dbe1cadde043a22d9d4efbea